### PR TITLE
Better navigation entry

### PR DESCRIPTION
**Pull request by nickvergessen**
@MorrisJobke @LukasReschke @rullzer @blizzz

### CODE REVIEW

File: appinfo/app.php [hunk 1]
```diff
@@ -15,20 +15,15 @@
...
-	// will it be listed in the navigation
-	'order' => 10,
-	// the route that will be shown on startup
-	'href' => \OCP\Util::linkToRoute('bookmarks.web_view.index'),
-	// the icon that will be shown in the navigation
-	// this file needs to exist in img/
-	'icon' => \OCP\Util::imagePath('bookmarks', 'bookmarks.svg'),
-	// the title of your application. This will be used in the
-	// navigation or on the settings page of your app
-	'name' => \OC_L10N::get('bookmarks')->t('Bookmarks')
```
Comment (nickvergessen) [inline appinfo/app.php]: :zap:

**Comment by rullzer** [comment]
:+1:

**Comment by MorrisJobke** [comment]
:+1:

**Comment by MorrisJobke** [comment]
cc @blizzz

**Pull request merged**